libs/vscode-ui/feature-task-execution-form/src/lib/task-execution-form.component.ts

```diff
@@ -249,11 +249,12 @@ export class TaskExecutionFormComponent implements OnInit, AfterViewChecked {
         );
         validators.push((control) => {
           if (
-            control.value &&
-            !validValueSet.has(control.value) &&
+            (control.value &&
+              !Array.isArray(control.value) &&
+              !validValueSet.has(control.value)) ||
             // multiselect values are Array, check if all values are in Set
-            control.value.length &&
```

Comment:
whoops, this was quite right. A valid control.value is either a string in the validValueSet or an array of values from validValueSet.